packages/server/src/app.ts

```diff
@@ -48,6 +49,9 @@ export default function(options: TAppOptions) {
       }
     });
   }
+
+  if (dev) app.use(morgan("dev"));
```

Comment:
I still don't grasp why your conditional middleware for the past logger wasn't working >.<